### PR TITLE
AAP 2.5 (4) Async Release-Updated Release Notes

### PR DESCRIPTION
**Pull request by rogrange**
These release notes cover the current enhancements and bug fixes for the AAP 2.5 (4) release on November 20, 2024.

Resolves: AAP-35387

### CODE REVIEW

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+== Bug fixes
+
+=== General
+
+With this update, the following CVEs have been addressed:
+
+link:https://access.redhat.com/security/cve/cve-2024-9902[CVE-2024-9902] ansible-core: Ansible-core user may read/write unauthorized content.
+
+link:https://access.redhat.com/security/cve/cve-2024-8775[CVE-2024-8775] ansible-core: Exposure of sensitive Information in Ansible vault files due to Improper logging.
```
Comment (sayjadha) [inline downstream/titles/release-notes/async/aap-25-4-20-nov.adoc:20]: 'Information' and 'Improper' should be in small cases.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
+[[aap-25-4-20-nov]]
+
+= {PlatformNameShort} patch release November 20, 2024
+
+The following enhancements and bug fixes have been implemented in this release of {PlatformNameShort}.
+
+== Enhancements
+
+* With this release, a redirect page has now been implemented that will be exhibited when you navigate to the root ``/`` for each component's stand-alone URL. The API endpoint remains functional. This affects {EDAName}, {ControllerName}, {OperatorPlatformNameShort}, and {OCPShort}.
```
Comment (sayjadha) [inline downstream/titles/release-notes/async/aap-25-4-20-nov.adoc:9]: **``/``** -> **`/`** (remove the extra backticks)

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+With this update, the following CVEs have been addressed:
+
+link:https://access.redhat.com/security/cve/cve-2024-9902[CVE-2024-9902] ansible-core: Ansible-core user may read/write unauthorized content.
+
+link:https://access.redhat.com/security/cve/cve-2024-8775[CVE-2024-8775] ansible-core: Exposure of sensitive Information in Ansible vault files due to Improper logging.
+
+
+=== {PlatformNameShort}
+
+* Fixed an issue where the user is unable to filter out hosts on inventory groups where it returns a *Failed to load* options on {PlatformNameShort} UI.
```
Comment (sayjadha) [inline downstream/titles/release-notes/async/aap-25-4-20-nov.adoc:25]: **is** -> **was**
**returns** -> **returned**
Past tense for fixed issues. This is a global comment.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+=== {OperatorPlatformNameShort}
+
+* Fixed a syntax error when `bundle_cacert_secret` is defined due to incorrect indentation.
+
+* Fixed an issue where the default operator catalog for {PlatformNameShort} aligns to cluster-scoped versus namespace-scoped.
+
+* Added the ability to set tolerations and `node_selector` for the Redis *statefulset* and the gateway deployment.
+
+* Ensure the platform URL status is set when *Ingress* is used to resolve an issue with {Azure} on Cloud managed deployments, where the {PlatformNameShort} operator fails to finish because it is looking for {OCPShort} routes that are not available on Azure Kubernetes Service.
```
Comment (sayjadha) [inline downstream/titles/release-notes/async/aap-25-4-20-nov.adoc:39]: It's too long a sentence; consider breaking it down for clarity.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+* Fixed an issue where the default operator catalog for {PlatformNameShort} aligns to cluster-scoped versus namespace-scoped.
+
+* Fixed an issue where the {PlatformNameShort} Operator description does not render code block correctly.
+
+* It is necessary to specify the `CONTROLLER_SSO_URL` and `AUTOMATION_HUB_SSO_URL` settings in Gateway to fix the OIDC auth redirect flow.
+
+* It is necessary to set the `SERVICE_BACKED_SSO_AUTH_CODE_REDIRECT_URL` setting to fix the OIDC auth redirect flow.
+
+=== Container-based {PlatformNameShort}
```
Comment (sayjadha) [inline downstream/titles/release-notes/async/aap-25-4-20-nov.adoc:49]: Do we say 'container-based AAP' or 'container-based installation'? Same for RPM. If it's the latter, then we need to use the attribute `{ContainerBase}`.

**Review by sayjadha — COMMENTED** [review]
@rogrange: A few minor comments, rest LGTM.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc (hunk 1 above)
Comment (rogrange) [reply downstream/titles/release-notes/async/aap-25-4-20-nov.adoc]: Done.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc (hunk 5 above)
Comment (rogrange) [reply downstream/titles/release-notes/async/aap-25-4-20-nov.adoc]: Done.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc (hunk 3 above)
Comment (rogrange) [reply downstream/titles/release-notes/async/aap-25-4-20-nov.adoc]: Done.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc (hunk 4 above)
Comment (rogrange) [reply downstream/titles/release-notes/async/aap-25-4-20-nov.adoc]: Done.

**Review by sayjadha — COMMENTED** [review]
Two minor comments.

File: downstream/titles/release-notes/async/aap-25-4-20-nov.adoc [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+* Fixed an issue where the default operator catalog for {PlatformNameShort} aligns to cluster-scoped versus namespace-scoped.
+
+* Fixed an issue where the {PlatformNameShort} Operator description does not render code block correctly.
+
+* It is necessary to specify the `CONTROLLER_SSO_URL` and `AUTOMATION_HUB_SSO_URL` settings in Gateway to fix the OIDC auth redirect flow.
+
+* It is necessary to set the `SERVICE_BACKED_SSO_AUTH_CODE_REDIRECT_URL` setting to fix the OIDC auth redirect flow.
+
+=== {ContainerBase} {PlatformNameShort}
```
[the remark this answers is not in the document]
Comment (rogrange) [reply downstream/titles/release-notes/async/aap-25-4-20-nov.adoc:47]: container-based installation Ansible Automation Platform'

**Comment by sayjadha** [comment]
Thank you for taking care of the suggested edits, @rogrange . Everything look's good so merging this ticket.

**Pull request merged**